superbench/benchmarks/micro_benchmarks/directx_gpu_copy_performance/Main.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <iostream>
+#include <sstream>
+
+#include "GPUCopyBw.h"
+
+int main(int argc, char *argv[]) {
+    Options *opts = new Options(argc, argv);
```

Comment:
> Options *opts = new Options(argc, argv); [](http://example.com/codeflow?start=3&length=41)

you are creating some objects in this file. It is OK for this application because the application quits when finishing its work, but I highly suggesting you to use smart pointer like unique_ptr to replace the raw pointers.

Reply:
updated, please help check it I use them in a correct way